helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -121,6 +126,7 @@
           .put(GlobalRebalancePreferenceKey.LESS_MOVEMENT, 1).build();
   private final static int MAX_REBALANCE_PREFERENCE = 10;
   private final static int MIN_REBALANCE_PREFERENCE = 0;
+  private final static boolean DEFAULT_ASYNC_GLOBAL_REBALANCE = true;
```

Comment:
DEFAULT_GLOBAL_REBALANCE_ASYNC_(MODE)_ENABLED? It reads better when the value is a boolean.